### ISSUE TITLE
Potential issue with cookies.

### ISSUE THREAD

**Issue by jaketame** [nosmokingbandit/Watcher3#88]
I'm using Docker Watcher and when using the login function it appears to set the cookie session but when exiting the browser and returning to watcher I am unable to login until I've cleared the cookies.

**Comment by nosmokingbandit** [comment]
I've suspected for a while that Docker was doing weird things with cookies. Unfortunately, every time I've attempted to set up Docker it has never worked correctly and my neighbors get concerned about the loud stream of profanities I scream at my computer.

I'll take another crack at getting Docker to cooperate just so I understand why the cookies/sessions aren't working the same as it does without Docker.
